examples/prefect/training.py

```diff
@@ -35,17 +37,21 @@ def train_linear_regression(X: np.ndarray, y: np.ndarray) -> base.BaseEstimator:
 
 @flow
 def prepare_regression_data(
-    random_state: int = 42, n_samples: int = 100, n_features: int = 1, noise: float = 0
+    random_state: int = 42, n_samples: int = 100, n_features: int = 1, noise: float = 0.2
 ) -> tuple[np.ndarray, ...]:
-    X, y = make_regression_data(random_state=random_state, n_features=n_features, noise=noise)
+    X, y = make_regression_data(
+        random_state=random_state, n_samples=n_samples, n_features=n_features, noise=noise
+    )
     X_train, y_train, X_test, y_test = make_train_test_split(X=X, y=y, random_state=random_state)
     return X_train, y_train, X_test, y_test
 
 
 @flow
-def prepare_regressor_and_test_data(
-    random_state: int = 42,
+async def prepare_regressor_and_test_data(
+    data_params: Union[dict[str, Union[int, float]], None] = None,
```

Comment:
If you're not using it in a typedef, could you use pipe syntax and `from __future__ import annotations` in line 1?